docs/en/SERVICE_CLIENTS.rst

```diff
@@ -646,6 +646,32 @@ Interface
     class ask_sdk_model.services.TemparatureUnit(Enum):
         CELSIUS = "CELSIUS"
         FAHRENHEIT = "FAHRENHEIT"
+Code Sample
+~~~~~~~~~~~
+**Alexa Customer Settings API**
+ 
+The Alexa Customer Settings API provides access to three pieces of information: preferred distance units, 
+preferred temperature units and the device's current configured time zone. When using the UpsServiceClient, 
+``get_system_distance_units`` and ``get_system_temperature_unit`` will return enum objects whose values must 
+be accessed by using the ``.value`` attribue. By comparision, ``get_system_time_zone`` will simply return a 
+string.
+ 
+.. code-block:: python
+ 
+    device_id = req_envelope.context.system.device.device_id
+    user_preferences_client = service_client_fact.get_ups_service()
```

Comment:
Can you change this to be called from `handler_input` directly, to avid confusion?
`handler_input.service_client_factory.get_ups_service()`

Reply:
Ah, of course. This has been updated in a new commit:
https://github.com/alexa/alexa-skills-kit-sdk-for-python/pull/79/commits/084acbebc5c0c9e510607537ab35b86ad74a3ab5